scapy/utils.py

```diff
@@ -1693,6 +1693,14 @@ def tcpdump(pktlist, dump=False, getfd=False, args=None,
             raise IOError("%s died unexpectedly !" % prog)
         else:
             proc.stdin.close()
+
+    stderr_output = proc.stderr.read()
```

Comment:
I don't think this is a good idea. If we may have to read from two file descriptors, I suppose we should use select() to do so, to make sure we cannot end in a dead lock.

Reply:
Actually, rather than parsing stderr and trying to figure out what happened, I think we should just raise an exception when tcpdump fails (and the message could include something like "use `quiet=False` to see tcpdump stderr` if `quiet` was set to `True`).